api/reference/services.adoc

```diff
@@ -246,13 +246,61 @@ POST /api/service_catalogs
 [[service-templates]]
 === Service Templates
 
-Basic Edits and Delete of Service Templates is available.
+Management of Service Templates is available via the following collection:
 
 [source,data]
 ----
 /api/service_templates
 ----
 
+Full CRUD is available on service_templates.
```

Comment:
Is there a reason for referencing service_templates rather than Service Templates? Inconsistent to some other "Full CRUD" references